repairs/public/js/warranty_claim.js

```diff
@@ -73,7 +73,21 @@ frappe.ui.form.on("Warranty Claim", {
 
 			// Start testing the item
 			if (frm.doc.status == "To Test") {
-				var repair_btn = frm.add_custom_button(__("Test Item"), () => {
+				let requires_scan = {
+					left_ear: false,
+					right_ear: false
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'right_ear' is not in camel case. (camelcase)](https://app.codacy.com/app/DigiThinkIT/repairs/pullRequest?prid=4722558)